pkg/koordlet/runtimehooks/runtimehooks.go

```diff
@@ -57,12 +57,14 @@ func (r *runtimeHook) Run(stopCh <-chan struct{}) error {
 		return err
 	}
 	if r.nriServer != nil {
-		if err := r.nriServer.Start(); err != nil {
-			// if NRI is not enabled or container runtime not support NRI, we just skip NRI server start
-			klog.Errorf("nri mode runtime hook server start failed: %v", err)
-		} else {
-			klog.V(4).Infof("nri mode runtime hook server has started")
-		}
+		go func() {
+			if err := r.nriServer.Start(); err != nil {
+				// if NRI is not enabled or container runtime not support NRI, we just skip NRI server start
+				klog.Errorf("nri mode runtime hook server start failed: %v", err)
```

Comment:
How about Warningf since the error is tolerated? Or we may distinguish the case in which the NRI should be started successfully but not.

Reply:
Yes, we could use Warningf as this error is not fatal.